modules/backup-etcd.adoc

```diff
@@ -14,26 +14,37 @@ Only save a backup from a single master host. Do not take a backup from each mas
 
 .Prerequisites
 
-* SSH access to a master host.
-
-.Procedure
-
-. Access a master host.
-
-. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have checked whether the cluster-wide proxy is enabled.
 +
 [TIP]
 ====
 You can check whether the proxy is enabled by reviewing the output of `oc get proxy cluster -o yaml`. The proxy is enabled if the `httpProxy`, `httpsProxy`, and `noProxy` fields have values set.
 ====
 
-. Run the `cluster-backup.sh` script and pass in the location to save the backup to.
+.Procedure
+
+. Start a debug session for a master node:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+. Change your root directory to the host:
 +
-Be sure to pass in the `-E` flag to `sudo` so that environment variables are properly passed to the script.
+[source,terminal]
+----
+sh-4.2# chroot /host
+----
+
+. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
```

Comment:
_If the cluster-wide proxy is enabled_
Isn't cluster-wide proxy being enables a prereq requirement?

Reply:
s/be sure/ensure? 
Ensure seems more common in the docs.

Reply:
No, actually it's not a requirement. They need to know up front whether the proxy is enabled, and if so, later in the procedure they'll have to perform a step.

We used to have the tip on how to check right in the procedure, but the procedure is changing so that they can't check easily at the place they are at the time, so that's why I moved it to a prereq.